spread.js

```diff
@@ -0,0 +1,11 @@
+
+const heading = document.querySelector(".show");
+  //console.log(heading);
+  heading.innerHTML = animateHeading(heading.textContent);
+
+function animateHeading (str) {
+  var wordArray = [...str].map(letter => `<span>${letter}</span>`).join('');
+  return wordArray;
+  console.log(wordArray);
```

Comment:
Code after return statement doesn't work
So, this console log will never execute